e2e/package.json

```diff
@@ -18,6 +18,7 @@
     "@types/node": "^11.13.4",
     "@types/rimraf": "^2.0.2",
     "@types/selenium-webdriver": "^3.0.16",
+    "axios": "^0.19.0",
```

Comment:
According to this list(<https://dev.eclipse.org/ipzilla/buglist.cgi?query_format=advanced&short_desc_type=allwordssubstr&short_desc=axios&long_desc_type=substring&long_desc=&bug_file_loc_type=allwordssubstr&bug_file_loc=&keywords_type=allwords&keywords=&emailassigned_to1=1&emailtype1=substring&email1=&emailassigned_to2=1&emailreporter2=1&emailcc2=1&emailtype2=substring&email2=&bugidtype=include&bug_id=&chfieldfrom=&chfieldto=Now&chfieldvalue=&cmdtype=doit&order=Reuse+same+sort+as+last+time&field0-0-0=noop&type0-0-0=noop&value0-0-0=>), we have CQ only for "0.17.1" axios version, so please specify this version or do CQ for your version.

Reply:
changed on 0.17.1